app/src/components/main/analytics/AnalyticsWrapper.jsx

```diff
@@ -77,7 +76,6 @@ const getAppVersion = (buildVersion) =>
         pattern_analysis: normalizeDimensionValue(isPatternAnalyzerEnabled),
         project_id: projectId,
         timestamp: Date.now(),
-        is_admin: isAdmin,
```

Comment:
Also you should do it for the Google Analytics Universal

Reply:
We did not send is_admin dimension for GA Universal.
In this case we just use isAdmin variable in order to determine whether we should send projectID or not.